RELEASE_NOTES.txt

```diff
@@ -27,7 +38,7 @@ NEW FEATURES:
 - Updates to the testing environment
 - "kb-sdk init" now generates a RELEASE_NOTES.md file in new module
 
-VERSION: 0.1.1 through 0.1.16 ( 11/19-2015 through 5/17/2017)
+VERSION: 1.0.1 through 1.0.16 ( 11/19-2015 through 5/17/2017)
```

Comment:
Was this just a mistake/typo people made in previous updates?

Reply:
No, https://semver.org/#spec-item-4

Reply:
this should not be changed as the numbers are built into the codebase

Reply:
fair enough then

Reply:
That's what I figured.